packages/wonder-blocks-icon-button/src/components/__tests__/icon-button-unstyled.test.tsx

```diff
@@ -386,4 +436,42 @@ describe("IconButton", () => {
             }).not.toThrow();
         });
     });
+
+    describe("Accessibility", () => {
+        describe("axe", () => {
+            it("should not have violations", async () => {
+                // Arrange
+                // Act
+                const {container} = render(
+                    <IconButtonUnstyled aria-label="Search">
+                        <PhosphorIcon icon={magnifyingGlassIcon} />
+                    </IconButtonUnstyled>,
+                );
+
+                // Assert
+                await expect(container).toHaveNoA11yViolations();
+            });
+        });
+
+        describe("ARIA", () => {
```

Comment:
One other a11y test I normally include is when a component accepts AriaProps, I also add a test for another aria attribute that isn't based on other props so that it confirms the `otherProps` are applied to the correct element!

Reply:
ohh interesting, do you have an example of that one in the current tests?

Reply:
Added that test here: https://github.com/Khan/wonder-blocks/pull/2624/commits/7ec5238dfb1bf14dcd4d9005a4d331e89addcf18